pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "gowermetric"
+version = "0.1.0"
+description = ""
+authors = ["Arch4ngel21 <adammichal657@gmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+numpy = "^1.23.3"
+black = "^22.8.0"
```

Comment:
1. Move black to dev dependencies: https://python-poetry.org/docs/master/managing-dependencies/#optional-groups
2. I personally prefer wildcard syntax, e.g. `numpy = "1.23.*"`. I think it is more explicit and similar to `requirements.txt`
3. Add pre-commit hook (https://pre-commit.com/) with black (https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/) to never forget about code formatting